From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Julien Massot <julien.massot@collabora.com>
Date: Thu, 6 Mar 2025 08:10:24 +0100
Subject: include: configs: mt8195: set scsi as first boot target

If scsi is enabled then it's most likely our first boot target
to check.

order become:
- ufs (if available)
- emmc / sdcard
- usb

It speedup the boot on ufs, and have no effect on Genio eMMC evks
since SCSI is disabled.

Signed-off-by: Julien Massot <julien.massot@collabora.com>
---
 include/configs/mt8195.h | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/include/configs/mt8195.h b/include/configs/mt8195.h
index 111111111111..222222222222 100644
--- a/include/configs/mt8195.h
+++ b/include/configs/mt8195.h
@@ -121,9 +121,10 @@
 #endif
 
 #define BOOT_TARGET_DEVICES(func) \
-	BOOT_TARGET_MMC(func) \
-	BOOT_TARGET_USB(func) \
-	BOOT_TARGET_SCSI(func)
+	BOOT_TARGET_SCSI(func) \
+	BOOT_TARGET_MMC(func)  \
+	BOOT_TARGET_USB(func)
+
 
 #if !defined(CONFIG_EXTRA_ENV_SETTINGS)
 #if !IS_ENABLED(CONFIG_SPI_FLASH)
-- 
Armbian

